test/extended/image_ecosystem/jenkins_plugin.go

```diff
@@ -665,6 +665,21 @@ var _ = g.Describe("[image_ecosystem][jenkins][Slow] openshift pipeline plugin",
 			ginkgolog("\n\nJenkins logs>\n%s\n\n", logs)
 			o.Expect(err).NotTo(o.HaveOccurred())
 
+			g.By("get build and confirm trigger by field is correct")
+			out, err := oc.Run("get").Args("builds/frontend-1", "-o", "jsonpath='{.spec.triggeredBy[0].message}'").Output()
+			o.Expect(err).NotTo(o.HaveOccurred())
+			o.Expect(strings.Contains(out, "Jenkins job")).To(o.BeTrue())
+
+			jobName = "test-build-clone-job"
+			data = j.readJenkinsJob("build-job-clone.xml", oc.Namespace())
+			j.createItem(jobName, data)
+			jmon = j.startJob(jobName)
+			jmon.await(10 * time.Minute)
+			g.By("get clone build console logs and see if succeeded")
```

Comment:
ok / cool / wanted to make sure we were on the same page (regarding where the values are coming from)

so yes, I'll work on both 1) (the idea I mentioned in my last comment) and 2) (agreed /  a good idea to add )